tests/unit/ui/components/TestPermission.js

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import Icon from 'ui/components/Icon';
+import Permission from 'ui/components/Permission';
+
+
+describe('Permission component', () => {
+  const defaultProps = {
+    className: 'my-class',
+    description: 'A decription',
+    key: 'someKey',
```

Comment:
Oops. I need to remove this. I've already added a bunch of comments to this PR, and if I push a new commit I think they'll get hidden, so just assume I'll fix this for now. :P